### PR TITLE
[PLAT-1570] -- opaque selection material

### PR DESCRIPTION
**Pull request by gbiv**
## Summary

Set opacity on default materials to 255
Set glossiness, ie specular exponent, to 50 (is this too glossy?)

## Test Plan

Verified in a test app
Will need to be picked up by Connect+ 

## Dependencies

[Ticket](https://vertexvis.atlassian.net/browse/PLAT-1570)

## NOTES
- [x] ~~Is 50 too high?~~ Set to 4
- [x] Does the version get bumped automatically or does it need to be specified in this PR
- [ ] Need to pick this up in Connect+

### CODE REVIEW

File: packages/viewer/src/lib/scenes/__tests__/scene.spec.ts [hunk 1]
```diff
@@ -294,7 +294,7 @@ describe(Scene, () => {
                       g: 0,
                       r: 0,
                     },
-                    ns: 10,
+                    ns: 50,
```
Comment (mikemccarty-vertex) [inline packages/viewer/src/lib/scenes/__tests__/scene.spec.ts:297]: RWS defaults to [4 when all else fails](https://github.com/Vertexvis/rendering-worker-service/blob/6eaa9442088f96d08a2e5c405b8415d82743b42f/src/Embree.cpp#L42).  There's no science behind that number, just something being better than nothing.

Shininess is expressed as an exponent so the effect is asymptotic as the values get larger.

File: packages/viewer/src/lib/scenes/colorMaterial.ts [hunk 2]
```diff
@@ -13,8 +13,8 @@ export interface ColorMaterial {
 }
 
 const defaultColor: ColorMaterial = {
-  opacity: 100,
-  glossiness: 10,
+  opacity: 255,
+  glossiness: 50,
```
Comment (mikemccarty-vertex) [inline packages/viewer/src/lib/scenes/colorMaterial.ts:17]: Can we pull the default glossiness out as a constant to ensure it is the same everywhere it is used?

Comment (gbiv) [reply packages/viewer/src/lib/scenes/colorMaterial.ts]: exported the defaultColor referenced the value from there.

File: packages/viewer/src/lib/scenes/__tests__/scene.spec.ts (hunk 1 above)
Comment (gbiv) [reply packages/viewer/src/lib/scenes/__tests__/scene.spec.ts]: Makes sense to me to sync it up with rw.

**Review by mikemccarty-vertex — APPROVED** [review]
LGTM

**Review by jdm717 — APPROVED** [review]

**Pull request merged**